backend_new/README.md

```diff
@@ -27,3 +27,90 @@ We use the following conventions:
   <li> a model's fields are <b>@map()</b>ed to lowercase snackcased (e.g. hello_world) </li>
 </ul>
 This is to make the api easier to work with, and to respect postgres's name space conventions
+<p></p>
+
+# Controllers
+Controllers are where backend endpoints are housed. They follow the [Nestjs standdards](https://docs.nestjs.com/controllers)
+
+They are housed under `/src/controllers` 
+
+## Conventions
+Controllers are given the extension `.contoller.ts` and the model name (listing, application, etc) is singular. So for example `listing.controller.ts`
+
+The exported class should be in capitalized camelcase (e.g. `ListingController`)
+
+# DTOs
+Data Transfer Objects. These are how we flag what fields endpoints will take in, and what the form of the response from the backend will be. 
+
+We use the packages [class-transformer](https://www.npmjs.com/package/class-transformer) & [class-validator](https://www.npmjs.com/package/class-validator) for this
+
+They are housed under `src/dtos`, and are broken up by what model they are related too. There are also shared DTOs which are housed under the shared sub-directory
```

Comment:
Sorry if this is a nit, but I would love to be consistent in the documentation in terms of whether or not the sentences do or don't end in periods

Reply:
LOL I'm sorry I think because I mostly send IMs instead of actual write ups thats creeping in, I went through and I think I caught all of em